lib/widgets/gallery_widget.dart

```diff
@@ -20,61 +19,31 @@ class GalleryWidget extends StatefulWidget {
 }
 
 class _GalleryWidgetState extends State<GalleryWidget> {
-  final FocusNode _focusNode = FocusNode();
-
   @override
   void dispose() {
-    _focusNode.dispose();
     super.dispose();
   }
 
-  // Handles the key events from the Focus widget and updates the
-  // _message.
-  KeyEventResult _handleKeyEvent(FocusNode node, KeyEvent event) {
-    setState(() {
-      if (event is KeyDownEvent) {
-        if (event.logicalKey == LogicalKeyboardKey.arrowLeft) {
-          widget.gallery.focusPrevious();
-        } else if (event.logicalKey == LogicalKeyboardKey.arrowRight) {
-          widget.gallery.focusNext();
-        } else if (event.logicalKey == LogicalKeyboardKey.enter) {
-          widget.onImageClicked(widget.gallery.currentItemPath);
-        } else if (event.logicalKey == LogicalKeyboardKey.arrowUp) {
-          // ??
-        } else if (event.logicalKey == LogicalKeyboardKey.arrowDown) {
-          // ??
-        }
-      }
-    });
-    return KeyEventResult.ignored;
-  }
-
   @override
   Widget build(BuildContext context) {
-    return Focus(
+    return FocusScope(
       autofocus: true,
-      focusNode: _focusNode,
-      onKeyEvent: _handleKeyEvent,
       child: GridView.extent(
         primary: false,
         padding: const EdgeInsets.all(16),
         crossAxisSpacing: 10,
         mainAxisSpacing: 10,
         maxCrossAxisExtent: 200.0,
-        children: widget.gallery.items
-            .map(
-              (item) => GestureDetector(
-                onTap: () {
-                  widget.gallery.setCurrentItemPath(item.path);
-                  widget.onImageClicked(item.path);
-                },
-                child: GalleryImage(
-                  path: item.path,
-                  selected: item.path == widget.gallery.currentItemPath,
-                ),
+        children: [
+          for (final item in widget.gallery.items)
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: GalleryImage(
+                path: item.path,
+                autofocus: widget.gallery.items.first.path == item.path,
               ),
-            )
-            .toList(),
+            ),
```

Comment:
padding is technically unnecessary here, just had it there for debugging purposes